packages/dashboard-frontend/src/components/WorkspaceProgress/CreatingSteps/Initialize/index.tsx

```diff
@@ -149,6 +149,8 @@ class CreatingStepInitialize extends ProgressStep<Props, State> {
       throw new Error(
         'Could not resolve devfile from private repository because authentication request is missing a parameter, contains an invalid parameter, includes a parameter more than once, or is otherwise invalid.',
       );
+    } else if (errorCode === 'ssl_exception') {
```

Comment:
@vinokurig These changes required an additional test.

`packages/dashboard-frontend/src/components/WorkspaceProgress/CreatingSteps/Initialize/__tests__/index.spec.tsx`

```
...
test('`ssl_exception` error code', async () => {
    const searchParams = new URLSearchParams({
      [FACTORY_URL_ATTR]: factoryUrl,
      [ERROR_CODE_ATTR]: 'ssl_exception',
    });

    renderComponent(store, searchParams);

    await jest.advanceTimersByTimeAsync(MIN_STEP_DURATION_MS);

    const expectAlertItem = expect.objectContaining({
      title: 'Failed to create the workspace',
      children: expect.stringContaining(
        'SSL handshake failed. Please, contact the cluster administrator.',
      ),
      actionCallbacks: [
        expect.objectContaining({
          title: 'Click to try again',
          callback: expect.any(Function),
        }),
      ],
    });
    await waitFor(() => expect(mockOnError).toHaveBeenCalledWith(expectAlertItem));

    expect(mockOnNextStep).not.toHaveBeenCalled();
  });
  ...
```
Could you fix it?

Reply:
Done, thanks for the code snippet.